packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-dynamic-intervention-policy-group.vue

```diff
@@ -8,7 +8,7 @@
 			</aside>
 		</div>
 		<p>
```

Comment:
Not directly related to PR but small fix 

This section should be in a for loop similar to this https://github.com/DARPA-ASKEM/terarium/blob/main/packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-static-intervention-policy-group.vue#L104-L113

Reply:
Apparently we are not allowing for dynamics to have multiple anymore so this is irrelevant